Sources/ProcessEnv/Process+Parameters.swift

```diff
@@ -70,20 +75,24 @@ public extension Process {
             }
 
             if let cwdPath = newValue?.path {
-                self.currentDirectoryPath = cwdPath
+                self.currentDirectoryURL = URL(fileURLWithPath: cwdPath, isDirectory: true)
```

Comment:
This change has OS-compatibility implications. Was it required?

Reply:
It was only required to fix the deprecation warnings from the compiler. I can revert that